### ISSUE TITLE
Azure Push Notifications Support

### ISSUE THREAD

**Issue by tylerablake** [EddyVerbruggen/nativescript-plugin-firebase#1196]
Does this plugin allow receiving push notifications from Azure Notification Hub? Using the sample code for setting it up but I'm receiving some errors.

On getCurrentPushToken() - I get an undefined token back.

My app doesn't crash on launch, it logs "Registered for push" but doesn't receive push notifications.

I have set up the cert and uploaded it to an Azure Notification Hub and updated the info.plist file and app.entitlements file like suggested by the guide [here](https://github.com/EddyVerbruggen/nativescript-plugin-firebase/blob/master/docs/MESSAGING.md)

Any guidance would be great! Thanks so much!

**Comment by tylerablake** [comment]
What's your email, I'll try to send you the details for the repo there. Thanks for your help on this!

**Comment by EddyVerbruggen** [comment]
It's in my bio @ https://github.com/EddyVerbruggen - please only share if it's quick and easy to reproduce because I have zero time for anything really.

**Comment by tylerablake** [comment]
I added you as a collaborator on my repo, it is uploading now, if you just `tns run ios --bundle` then it should try to initialize push notifications inside of the app.component.ts file

**Comment by johny8228** [comment]
Hi,

I want to use the firebase plugin with Azure Notifications Hub? Is this plugin working for Android?
I followed the demo project and all required steps to link Azure Notifications Hub to Firebase but I don't receive any message.

Thanks,
Johny

**Comment by tylerablake** [comment]
@johny8228 Yes, I ended up using a combination of this plugin(to get the push token) and the `azure-mobile-apps` plugin (to register the push token with Azure). Are you on the NS slack? I can message you on there if you are.
